setup.sh

```diff
@@ -44,6 +44,16 @@ ln -fs $(pwd)/nginx-site /etc/nginx/sites-enabled/rovercode
 rm -rf /etc/nginx/sites-enabled/default > /dev/null
 service nginx restart
 
+read -p "Do you want to automatically start on boot? (choose N for development) :" answer
+case ${answer:0:1} in
+    y|Y )
+        sed -i -e '$i \bash '$(pwd)'\/start.sh \n' /etc/rc.local
+    ;;
+    n|N )
+        echo Not automatically starting on boot...
+    ;;
```

Comment:
You might want a default case `*)` in case the user doesn't put in `y` or `n`. Or just replace the `n|N` case and say that anything other than `y` is a no.